### DISCUSSION TITLE
Wipy end-of-life and pycom.io bankruptcy

### DISCUSSION THREAD

**Discussion by alvaroalvarezvazquez** [micropython/micropython#9676] [Hardware & Peripherals]
Just to let know to all members that might be affected. I was working with wipy devices since few years ago, and their micropython implementation, and many others might unknow the latest sad news. 

Pycom (company in charge of the Wipy devices) announced last July the end of life by the end of 2022 of all original ESP32 product based solutuions, leading to just 2 new products:
[F01 H7](https://pycom.io/product/f01-h7-oem-module/): ST M32 H7 chip + NXP secure element
[F01 S3](https://pycom.io/product/f01-s3-oem-module/): ESP32-S3 chip + NXP secure element
https://pycom.io/hardware-end-of-life-announcement/

BUT since one month ago, this is also reported at pycom.io main webpage:
PYCOM Limited – In Administration
On Friday 16 September 2022, Matthew Fox of Beacon LIP Limited was appointed Administrator in the High Court of Justice.
The Administrator seeks to achieve a sale of the business and assets of the company and is to keep existing customers supported during this time.
If you are interested in acquiring the business and assets of the company please contact independent agents, SIA Group and email [kevin.counihan@sia-group.co.uk](mailto:kevin.counihan@sia-group.co.uk).
For all other enquiries please contact [chanise@beaconlip.com](mailto:chanise@beaconlip.com) or [izzy@beaconlip.com](mailto:izzy@beaconlip.com)

Therefore most probably new pycom products won't be on market ever. I was waiting to F01, but I have started to migrate to other micropython board alternatives.

**Comment by scruss** [comment]
It's a shame they've gone away. In my past life at Elmwood Electronics, we couldn't get enough of the pycom boards: we sold them by the hundreds.

I suspect lack of availability for the F01's STM32 H7 chip may have done for them.

**Comment by robert-hh** [comment]
Quite a few things came together. Surely the lack of supply, but as well the dead horse PyGo, from which they did not climb off, the helium miner and the constant chaotic software development.

**Reply by scruss** [reply]
didn't know about the crypto stuff. I'll therefore add at least one "lol" to my response

**Comment by daanpape** [comment]
Hi,

I wanted to let you know that we have developed a pin compatible alternative which also adds lots of improvements (hardware reset, switchable 3.3Vout, ESP32-S3 based, GNSS built-in,...) And we don't work with a Micropython fork but with vanilla Micropython with a modem library. You can read all about it via https://www.crowdsupply.com/dptechnics/walter
